lib/widgets/enter_payment_info_dialog.dart

```diff
@@ -114,8 +114,11 @@ class EnterPaymentInfoDialogState extends State<EnterPaymentInfoDialog> {
                 validator: (value) {
                   if (parseNodeId(value) == null &&
                       _decodeInvoice(value) == null &&
-                      !isLightningAddress(value)) {
+                      !isLightningAddress(value) &&
+                      !isLNURL(value)) {
                     return texts.payment_info_dialog_error;
+                  } else if (isLNURL(value)) {
+                    widget.lnurlBloc.lnurlInputSink.add(value);
```

Comment:
This logic does not belong here, this functions only purpose is to validate if the input if valid or not.

The LNURL should be handled on pressing Approve.

Reply:
Agree, moved this logic. Please have a look at the latest rebase.